include/NovelRT/Timing/StepTimer.h

```diff
@@ -59,11 +57,11 @@ namespace NovelRT::Timing {
       _targetElapsedTicks = value;
     }
 
-    inline double getTargetElapsedSeconds() const {
+    inline Timestamp getTargetElapsedTime() const {
       auto targetElapsedTicks = getTargetElapsedTicks();
-      return TicksToSeconds(targetElapsedTicks);
+      return Timestamp(targetElapsedTicks);
     }
-    inline void setTargetElapsedSeconds(double value) {
+    inline void setTargetElapsedTime(double value) {
```

Comment:
This should take in a `Timestamp` not a `double`, so it correctly mirrors `getTargetElapsedTime`.

`SecondsToTicks` and `TicksToSeconds` should then likely be moved onto `Timestamp` instead of being here.

Reply:
Resolved as per the Discord discussion with new ctor.